packages/yourloops/components/device/models/device.models.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021-2022, Diabeloop
+ * Copyright (c) 2023, Diabeloop
```

Comment:
```suggestion
 * Copyright (c) 2021-2023, Diabeloop
```

Reply:
it's a new file, I think git is too high :D